components/num-util/examples/permyriad.rs

```diff
@@ -0,0 +1,13 @@
+// In computers, monetary values are sometimes stored as integers representing one ten-thousandth
```

Comment:
Should this comment be RustDoc?  Do the examples show up on docs.rs?

Reply:
nope.